services/transactionpool/test/add.go

```diff
@@ -46,4 +46,19 @@ var _ = Describe("transaction pool", func() {
 
 	})
 
+	It("dont forward an invalid transaction with gossip", func() {
+
+		tx := test.TransferTransaction().WithInvalidContent().Build()
+
+		gossip.When("BroadcastForwardedTransactions", mock.Any).Return(&gossiptopics.EmptyOutput{}, nil).Times(0)
```

Comment:
I'm not sure if Times(0) actually does anything. Did you see the test fail?

Reply:
Yup. If it happens more than 0 times the test fails